background/main.js

```diff
@@ -1,11 +1,22 @@
 var SERVER_URL = "https://chronicle.dev.mozaws.net";
 var CREATE_VISIT_ENDPOINT = SERVER_URL + "/v1/visits";
 
-function createVisit(historyItem) {
-  console.log('create visit', historyItem);
+function createVisit(tabId, changeInfo, tab) {
+  // Filter out some visits:
```

Comment:
Makes sense. Guidelines around this were a "quick and dirty disposable port" of the Firefox add-on, which didn't have any of these abstractions either. :)